README.md

```diff
@@ -25,269 +25,221 @@
 
-sidekiq-scheduler is an extension to [Sidekiq](http://github.com/mperham/sidekiq) that adds support
-for running scheduled jobs, which are like cron jobs, recurring on a regular basis.
+`sidekiq-scheduler` is an extension to [Sidekiq](http://github.com/mperham/sidekiq) that
+pushes jobs in a scheduled way, mimicking cron utility.
 
 ## Installation
 
-Add this to your Gemfile:
-
-``` ruby
-gem 'sidekiq-scheduler', '~> 2.0'
+``` shell
+gem install sidekiq-scheduler
 ```
 
-If you are using Rails you are set.
+## Usage
+
+### Hello World
 
-If you are not using Rails create a file with this content:
 
 ``` ruby
-# config/initializers/scheduler.rb
+# hello-scheduler.rb
 
 require 'sidekiq-scheduler'
 
-Dir[File.join(__dir__, '../../lib/workers/**/*.rb')].each(&method(:require))
+class HelloWorld
+  include Sidekiq::Worker
+
+  def perform
+    puts 'Hello world'
+  end
+end
 ```
 
-and then execute:
+``` yaml
+# config/sidekiq.yml
 
-``` sh
-sidekiq -r ./config/initializers/scheduler.rb
+:schedule:
+  hello_world:
+    every: 1m   # Runs once per minute
+    class: HelloWorld
 ```
 
-Look at [Loading the schedule](https://github.com/moove-it/sidekiq-scheduler/#loading-the-schedule)
-for information on how to load your schedule.
+Run sidekiq:
 
-You can add sidekiq-scheduler configuration options to `sidekiq.yml` config file.
-Available options are:
+``` sh
+sidekiq -r ./hello-scheduler.rb
+```
+
+You'll see the following output:
 
-``` yaml
-:schedule: <the schedule to be run>
-:dynamic: <if true the schedule can be modified in runtime [false by default]>
-:enabled: <enables scheduler if true [true by default]>
-:scheduler:
-  :listened_queues_only: <push jobs whose queue is being listened by sidekiq [false by default]>
 ```
+2016-12-10T11:53:08.561Z 6452 TID-ovouhwvm4 INFO: Loading Schedule
+2016-12-10T11:53:08.561Z 6452 TID-ovouhwvm4 INFO: Scheduling HelloWorld {"cron"=>"0 * * * * *", "class"=>"HelloWorld"}
+2016-12-10T11:53:08.562Z 6452 TID-ovouhwvm4 INFO: Schedules Loaded
 
-## Manage tasks from Unicorn/Rails server
+2016-12-10T11:54:00.212Z 6452 TID-ovoulivew HelloWorld JID-b35f36a562733fcc5e58444d INFO: start
+Hello world
+2016-12-10T11:54:00.213Z 6452 TID-ovoulivew HelloWorld JID-b35f36a562733fcc5e58444d INFO: done: 0.001 sec
 
-If you want start sidekiq-scheduler only from Unicorn/Rails, but not from Sidekiq you can have
-something like this in an initializer:
+2016-12-10T11:55:00.287Z 6452 TID-ovoulist0 HelloWorld JID-b7e2b244c258f3cd153c2494 INFO: start
+Hello world
+2016-12-10T11:55:00.287Z 6452 TID-ovoulist0 HelloWorld JID-b7e2b244c258f3cd153c2494 INFO: done: 0.001 sec
+```
 
-``` ruby
-# config/initializers/sidekiq_scheduler.rb
-require 'sidekiq/scheduler'
+## Configuration options
 
-puts "Sidekiq.server? is #{Sidekiq.server?.inspect}"
-puts "defined?(Rails::Server) is #{defined?(Rails::Server).inspect}"
-puts "defined?(Unicorn) is #{defined?(Unicorn).inspect}"
+Configuration options are placed inside `sidekiq.yml` config file.
```

Comment:
I'd clarify that they are placed here by default so that it leads to the part where you explain how to use a different config file.